### ISSUE TITLE
Mass sliders not working with shift key in built downloaded version

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#108]
This issue may be related to https://github.com/phetsims/gravity-force-lab/issues/105. 

I am working offline, so yesterday I downloaded a built version from master. I noticed that the mass sliders do not advance when I try to use the shift key to take small steps on the mass sliders. 

@jessegreenberg and @mbarlow12 tagging you both, so you can determine who should be assigned. This is not an urgent issue for me, but it would be good to check a built downloaded version of GFLB as this issue may be a common code issue.

**Comment by mbarlow12** [comment]
@terracoda I just created a fresh build from master and it seems to be working ok for me, so it may be a bug that was fixed more recently. I'll keep this open so that we can check when we have a new dev release. Thanks for the heads up.

**Comment by jessegreenberg** [comment]
Hey I think this may be related to https://github.com/phetsims/scenery-phet/issues/411.  I agree this has been fixed, and can be closed if @mbarlow12 and @terracoda agree.

**Comment by terracoda** [comment]
I downloaded a built version and the sliders now work great! Closing.
